tests/pipelines/stable_diffusion/test_stable_diffusion.py

```diff
@@ -46,39 +46,11 @@
 
 
 class StableDiffusionPipelineFastTests(PipelineTesterMixin, unittest.TestCase):
-    def tearDown(self):
-        # clean up the VRAM after each test
-        super().tearDown()
-        gc.collect()
-        torch.cuda.empty_cache()
+    pipeline_class = StableDiffusionPipeline
```

Comment:
Additional attributes for each fast test class:
* `pipeline_class` to use in the common tests from `PipelineTesterMixin`
* `get_common_pipeline_components()` to generate the dummy pipeline components (unet, text_encoder, etc.) both for the mixin and the custom pipeline tests
* `get_common_inputs(device, seed=0)` to generate dummy inputs for the tests

Reply:
Just one stupid question, what does `common` refer to here? Would it be clearer if we say `get_pipeline_components()` and `get_pipeline_inputs()` (or `get_inputs()`)

Reply:
@pcuenca `common` was meant to signal that it's used by the mixin as well, but `get_pipeline_components` or `get_dummy_pipeline_components` should be ok too

Reply:
Understood, thanks.